### PR TITLE
17.0 fix change costing method arm

### PR DESCRIPTION
**Pull request by amoyaux**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Whenrow** [comment]
robodoo r+ rebase-ff

**Comment by amoyaux** [comment]
robodoo r+ rebase-ff

**Comment by Whenrow** [comment]
robodoo r+ rebase-ff

File: addons/stock_account/tests/test_stockvaluation.py [hunk 1]
```diff
@@ -4374,6 +4396,66 @@ def test_journal_entries_from_change_product_cost_method(self):
...
+        self.assertRecordValues(
+            amls,
+            [
+                {'account_id': self.stock_valuation_account.id, 'debit': 1071.0, 'credit': 0.0},
+                {'account_id': self.stock_output_account.id, 'debit': 0.0, 'credit': 1071.0},
+                {'account_id': self.stock_output_account.id, 'debit': 1071.0, 'credit': 0.0},
+                {'account_id': self.stock_valuation_account.id, 'debit': 0.0, 'credit': 1071.0},
+            ]
+        )
+
```
Comment (Williambraecky) [inline addons/stock_account/tests/test_stockvaluation.py:4458]: Hi 👋 

This test is failing without demo data: https://runbot.odoo.com/runbot/build/71177078
Please fix it in subsequent FW-Ports and backport the fix in 17.0.

@amoyaux @Whenrow

Comment (Whenrow) [reply addons/stock_account/tests/test_stockvaluation.py:4458]: @Williambraecky sorry about that. I'm fixing it

Comment (Williambraecky) [reply addons/stock_account/tests/test_stockvaluation.py:4458]: Thanks :)

**Pull request closed without merging**